mlrun/runtimes/databricks_job/databricks_wrapper.py

```diff
@@ -44,6 +44,28 @@ def get_task(databricks_run: Run) -> RunTask:
     return databricks_run.tasks[0]
 
 
+def log_artifacts_by_dbfs_json(
+    context: mlrun.MLClientCtx,
+    workspace: WorkspaceClient,
+    artifact_json_path: str,
+    databricks_run_name: str,
+):
+    if not workspace.dbfs.exists(artifact_json_path):
+        return
+    context.logger.info(f"Artifacts found. Run name: {databricks_run_name}")
+    with workspace.dbfs.open(artifact_json_path, read=True) as artifact_file:
+        artifact_json = json.load(artifact_file)
+    for artifact_name, artifact_path in artifact_json.items():
```

Comment:
This is where you may fail because you didn't verify the contents of the json are strings - this may cause the runtime code to explode if the user provided other things such as dicts or whatever.

Reply:
add a type decorator in mlrun_log_artifact. tnx!